group-owner.html

```diff
@@ -0,0 +1,299 @@
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h2><a class="anchor" id="add"></a>Adding a user to your group</h2>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h3>OMERO.web</h3>
+
+<div class="line-block">
+  <div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<ol class="step">
+
+<li><p class="first">Click on the menu with your name on the right in the top toolbar.</p>
+
+<p>Select <span class="bold">User Settings</span> in the menu.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/groupOwnerEditGroup.jpg" class="align-center" src="images/groupOwnerEditGroup.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+<li><p class="first">Click on the <span class="bold">My Groups</span> tab.</p>
+
+<p>Click on the <span class="bold">Edit</span> button.</p>
+
+<p>Click into the <span class="bold">Members</span> or <span class="bold">Owners</span> text box and start typing a user's name, select the user from the list, and click or hit return to add them to the group.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/groupOwnerSettings.jpg" class="align-center" src="images/groupOwnerSettings.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+</ol>
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h2><a class="anchor" id="remove"></a>Removing a user from your group</h2>
+
+<ol>
+
+<h3>OMERO.web</h3>
+
+<li><p class="first">Click on the menu with your name on the right in the top toolbar.</p>
+
+<p>Select <span class="bold">User settings</span> in the menu.</p>
+
+<p>Click on the <span class="bold">My Groups</span> tab.</p>
+
+<p>Click on the <span class="bold">Edit Group</span> button.</p>
+
+<p>Click on the <span class="bold">X</span> to the right of the users name in the members or owners list.</p>
+
+<p>Click <span class="bold">Save</span> to complete the change.</p>
+
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/facilityManagerRemoveUser.jpg" class="align-center" src="images/facilityManagerRemoveUser.jpg" style="width: 100%;" />
```

Comment:
This screenshot shows removing someone by editing the user but the text above describes removing them via the edit group option instead.